src/components/message/MessageReplayModal.tsx

```diff
@@ -49,6 +50,8 @@ function MessageReplayModal() {
 
 	const [isOpen, setIsOpen] = React.useState(false);
 
+	const [draggable, setDragable] = React.useState(true);
```

Comment:
this doesn't work when you drag vertically.

you probably need this https://www.framer.com/docs/gestures/###dragcontrols